src/components/IconSet.vue

```diff
@@ -177,13 +177,13 @@ onKeyStroke('Escape', () => {
           <form action="/collection/all" class="flex-auto" role="search" method="get" @submit.prevent>
             <input
               ref="input"
-              :value="search"
+              v-model="search"
```

Comment:
I see you reverted this change. This is mentioned in PR description, you can see in my screen recording I have my IME activated and the search still works. Without this change I have to submit the candidate everytime.

This is especially inconvenient on mobile, where even people using English type in IMEs (Gboard etc.) often.